config/settings/base.py

```diff
@@ -67,3 +76,40 @@
 
 
 STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {
+    # Use hyperlinked styles by default.
+    'DEFAULT_MODEL_SERIALIZER_CLASS':
+        'rest_framework.serializers.ModelSerializer',
+    'DEFAULT_PERMISSION_CLASSES': (
+        # 'roles.permissions.MFLPermission',
+        'rest_framework.permissions.AllowAny',
+
+    ),
+    'DEFAULT_FILTER_BACKENDS': (
+        'rest_framework.filters.DjangoFilterBackend',
+    ),
+    'DEFAULT_PARSER_CLASSES': (
+        'djangorestframework_camel_case.parser.CamelCaseJSONParser',
```

Comment:
I am ambivalent about the use of camel case; since the front-end has not started, let us leave the parsers and renderers at the default "underscoreized" JSON format.

Also:

 * add in XML parser / renderer
 * add a CSV renderer: https://github.com/mjumbewu/django-rest-framework-csv
 * add a Pandas ( Excel ) renderer: https://github.com/wq/django-rest-pandas

The first one is for compatibility purposes; the last two will be the primary mechanism for export of data ( export will be via a custom DRF serializer, with all applicable DRF filters being usable ).